src/main/java/uk/gov/hmcts/reform/demo/controllers/RootController.java

```diff
@@ -23,6 +35,29 @@
      */
     @GetMapping("/")
     public ResponseEntity<String> welcome() {
-        return ok("Welcome to spring-boot-template");
+        return ok("Welcome to pip-business-rules");
+    }
```

Comment:
Replace with healthcheck?

Reply:
So I think this is needed, and I've checked another project (non PIP), which has also kept it in too, although without the message.

Although I'm not familiar with Azure - The Java Doc suggests that Azure requires something to be on the root endpoint when AlwaysOn mode is turned on, therefore I'm thinking for now we want to keep it in.

The health check seems to be running on '/health'